test/cpp/end2end/xds/xds_cluster_end2end_test.cc

```diff
@@ -1515,6 +1610,11 @@ TEST_P(ClientLoadReportingTest, BalancerRestart) {
   EXPECT_EQ(0U, client_stats.total_requests_in_progress());
   EXPECT_EQ(0U, client_stats.total_error_requests());
   EXPECT_EQ(0U, client_stats.total_dropped_requests());
+  ASSERT_THAT(
+      client_stats.locality_stats(),
+      ::testing::ElementsAre(::testing::Pair("locality0", ::testing::_)));
```

Comment:
Why not just replace the `::testing::_` with `::testing::IsEmpty()`, so that you don't need a separate expectation immediately below?

Reply:
Removed, will comment in the next PR and address there.